### PR TITLE
feat: Steps support clickable

### PR DESCRIPTION
**Pull request by zombieJ**
### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 👻 What's the background?

屏幕快照 2019-05-24 下午3 26 13

### 💡 Solution

Update `rc-steps` & styles.

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Steps support click to switch |
| 🇨🇳 Chinese | Steps 支持点击切换功能 |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/steps/demo/clickable.md](https://github.com/ant-design/ant-design/blob/steps-clickable/components/steps/demo/clickable.md)
[View rendered components/steps/index.en-US.md](https://github.com/ant-design/ant-design/blob/steps-clickable/components/steps/index.en-US.md)
[View rendered components/steps/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/steps-clickable/components/steps/index.zh-CN.md)

### CODE REVIEW

File: components/steps/demo/clickable.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    console.log('onChange:', current);
+    this.setState({ current });
+  };
+
+  render() {
+    const { current } = this.state;
+
+    return (
+      <div>
+        <Steps current={current} onChange={this.onChange}>
```
Comment (afc163) [inline components/steps/demo/clickable.md]: 文档可能要写明绑定 `onChange` 会变成 clickable。

**Review by afc163 — APPROVED** [review]

**Comment by afc163** [comment]
close #15182

**Comment by JonathanLee-LX** [comment]
onChange can use in latest version, but lack of type definition.

**Comment by afc163** [comment]
@JonathanLee-LX PR is welcome~

**Pull request merged**